README.md

```diff
@@ -19,6 +19,23 @@ running on GRiSP2. This includes:
 
 ## How to Build
 
+
+### Requirements
+
+#### macOS
+
+Building the toolchain on macOS requires a few dependencies:
+
+* `dtc`
```

Comment:
I think that is currently a multi platform dependency. Beneath that it should be a recent dtc that supports the -@ option (1.6 works, 1.4 doesn't).

That problem is known on the RTEMS mailing list too. There are some discussions to add a build receipe for rtems-source-builder for a dtc 1.6 after the outstanding RTEMS 5.1 release. There is only a problem with some dependencies on FreeBSD (libyaml) that won't be fixed before the release.

Reply:
Ah, got it. I just added what I saw was missing on macOS (there's probably more, like Command Line Tools for Xcode (`gcc` etc.). The intent was to add other OS requirements as we go (I can currently only test and provide feedback on macOS).

I don't understand the second paragraph. Is it enough if we list `dtc` 1.6 as a requirement for now?

Reply:
> Ah, got it. I just added what I saw was missing on macOS (there's probably more, like Command Line Tools for Xcode (`gcc` etc.). The intent was to add other OS requirements as we go (I can currently only test and provide feedback on macOS).

OK. We should just add a Linux paragraph too then with currently the same requirement (dtc 1.6 or newer).

> I don't understand the second paragraph. Is it enough if we list `dtc` 1.6 as a requirement for now?

What I wanted to say is that (with some luck) we can just build and install a dtc in the prefix environment in a few weeks. Prerequisite: Someone adds dtc to the rtems-source-builder which won't happen before the outstanding RTEMS 5.1 release. After the release it's a hobby topic. So it doesn't have a fixed schedule.

Reply:
Once #3 passes I'll add Linux info to the README as well.